### ISSUE TITLE
Handling of Diplonema in OTT3.0

### ISSUE THREAD

**Issue by mtholder** [OpenTreeOfLife/reference-taxonomy#304]
There is a lot more merging of taxa in OTT3.0. Contrast v3.0:

 https://devtree.opentreeoflife.org/taxonomy/browse?name=diplonema

and

https://tree.opentreeoflife.org/taxonomy/browse?name=diplonema

The merging of genus Hybocodon (https://tree.opentreeoflife.org/taxonomy/browse?id=4717850) into diplonema (excavata) in OTT3.0 is problematic in that it results in Metazoa not being monophyletic in the synth tree

**Comment by jar398** [comment]
There are a lot of names in this situation, probably about a thousand, where the separation taxon of one is a child of the separation taxon of the other (e.g. one in Metazoa, the other in Eukaryota). Many of these should be merged (want no homonym), because the taxon is misclassified or incertae sedis in a lower priority taxonomy, but many of them shouldn't, e.g. Diplonema (want homonym). I don't know how to tell, but I may be able to come up with something, such as creating additional separation taxa. It will require scrutinizing a sampling of instances.

I may have changed the default in these cases from no match to match. If I did so it was to try to get rid of a large number of duplicate taxa. (Or I may misremember and there is another explanation.) I guess I should change it back. Not-merging is always safer than merging, since a merge can always be done later on but there is no such thing as an unmerge.

**Comment by jar398** [comment]
Did some more digging.  Changing the heuristic does in fact fix Diplonema, and probably at least 70 others.  I looked at a sampling of 10 of the 228 cases where the heuristic in question makes the alignment decision. Of the ten, 3 are correctly split and 7 are incorrectly split (i.e. should have been merged). (So in draft3, I would expect 3 to be incorrectly merged, and 7 correctly merged.)

So, while reverting to 2.10 behavior increases the overall number of errors, it results in incorrect splits instead of incorrect merges. Bad splits (homonyms) are nuisance, but easier to deal with than bad merges. So reverting is the fix I plan to do.
